src/components/ProductCard/ProductCard.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import './ProductCard.scss';
+
+type Product = {
+  images: string;
```

Comment:
In this type, as I assume, we are going to need an 'id' identifier for mapping products in the future. But I created the Product type for the special ‚get' function. So it's not a problem. Just saying)